rts/lra/trip-controller/src/main/java/io/narayana/rts/lra/demo/tripcontroller/TripController.java

```diff
@@ -220,5 +223,24 @@ private Booking bookFlight(String flightNumber, String bookingId) throws Booking
 
         return response.readEntity(Booking.class);
     }
+
+    @PUT
+    @Path("/after")
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
I don't want to sound picky, but it seems that you are not returning any entity from this method so this may be misleading?

Reply:
I will remove it.